src/content-helper/post-list-stats/class-post-list-stats.php

```diff
@@ -296,12 +299,12 @@ public function get_parsely_stats_response( $analytics_api ) {
 			return null;
 		}
 
-		$response = $analytics_api->get_posts_analytics(
+		$response = $this->content_api->get_posts(
 			array(
-				'period_start'   => Analytics_Posts_API::ANALYTICS_API_DAYS_LIMIT . 'd',
+				'period_start'   => 'max_days',
```

Comment:
I wonder if we're introducing a "magic string" danger here for `period_start` and `limit` params.

Reply:
Yeah it's not ideal. I did tried a different approach here, to avoid using these magic strings directly. Let me know your thoughts.

Reply:
Maybe there could be ways to use the original constants, but I think this is fine as it is now. Might be simpler that way.